client/app/hearings/components/assignHearings/TimeSlotCard.stories.js

```diff
@@ -22,28 +23,8 @@ const Template = (args) => {
 
   return (
     <TimeSlotCard
-      hearing={{
-        regionalOfficeTimezone: args.regionalOfficeTimezone,
-        appellantFirstName: args.appellantFirstName,
-        appellantLastName: args.appellantLastName,
-        veteranFirstName: args.veteranFirstName,
-        veteranLastName: args.veteranLastName,
-        veteranFileNumber: args.veteranFileNumber,
-        aod: args.aod,
-        scheduledTimeString: args.scheduledTimeString,
-        issueCount: args.issueCount,
-        docketName: args.docketName,
-        caseType: args.caseType,
-        appealId: args.appealId,
-        requestType: args.requestType,
-        label: args.label,
-        docketNumber: args.docketNumber,
-        showDetails: args.showDetails,
-        showType: args.showType,
-        isFormerTravel: args.isFormerTravel,
-        itemSpacing: args.itemSpacing,
-        poaName: args.poaName
-      }}
+      hearing={defaultHearing}
```

Comment:
This works now, but the controls are not changing the time slot card, I think you can fix that by adding the following:

```suggestion
      hearing={{args || defaultHearing}
```